wallet/src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -875,4 +974,5 @@ instance Migrate WalletStorage where
         , _wsChangeAddresses = mapAddrKeys _v2_wsChangeAddresses
         }
       where
-        mapAddrKeys = HM.fromList . fmap (first unsafeCIdToAddress) . HM.toList
+        mapAddrKeys  = HM.fromList . fmap (first unsafeCIdToAddress) . HM.toList
+        migrateMapElements = HM.fromList . fmap (second migrate) . HM.toList
```

Comment:
You haven't introduced a new version here, instead tacking on an additional migration step between two existing versions. The problem occurs if anywhere has already migrated to the version using `Address`, because then they will not be able to migrate further and will have problems. Maybe the chance of that is sufficiently low (it's only been in develop for a couple of days, albeit in master for longer), but still, it would be better to have separate migrations here.